app/src/androidTest/java/com/github/sdp_begreen/begreen/fragments/SendPostFragmentTest.kt

```diff
@@ -233,7 +270,152 @@ class SendPostFragmentTest {
                 user.id
             )
         }
-
     }
 
+    /* This does not pass CI, eventhough it works locally. Still investiagting why
```

Comment:
I don't think its the cause, it seems like the problem is linked to retrieving the position of the User, and this could also explain why tests do not pass in `MapFragment`, but I don't know exaclty where the problem is.